vendor/k8s.io/code-generator/generate-groups.sh

```diff
@@ -1,92 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
the dep removes it. however, we need this file for our CRDs. please manually undelete.